src/main/java/net/pms/network/RequestV2.java

```diff
@@ -1182,6 +1183,12 @@ private StringBuilder browseSearchHandler(BrowseSearchRequest requestMessage) {
 			searchCriteria
 		);
 
+		for (DLNAResource dlnaResource : files) {
```

Comment:
We are already looping through `files` further down on line `1202`, can you add this logic to that existing loop for performance?

Reply:
good point. Have done it ...